client/dist/styles.css

```diff
@@ -1,53 +1,111 @@
 #app {
   background-color: lightgray;
   display: flex;
-  margin-left: auto;
-  margin-right: auto;
-}
 
-#imageModalImage #PanAndZoomImage-container{
-  margin: 0;
-  background: black;
-  position: fixed;
-  top: 0%;
-  left: 0%;
-  transition: all .2s ease;
-  /* padding: 100% */
 }
 
-#imageModalBG {
+#ImageModal{
```

Comment:
Jake, from what I know it is recommended to not use camelCase for naming classes and ids in CSS. I'm not sure how big of a deal it is and if it's necessary to rename your classes and ids so they are all lower-case at this point, but it's something to keep in mind for the future projects.